build.gradle

```diff
@@ -1,16 +1,16 @@
 ext {
   kotlinVersion = '1.0.3'
-  antlrVersion = '4.5'
+  antlrVersion = '4.5.3'
 
   compileSdkVersion = 24
-  buildToolsVersion = '24.0.0'
+  buildToolsVersion = '24.0.2'
 
   dep = [
-    androidPlugin: 'com.android.tools.build:gradle:1.5.0',
-    supportAnnotations: 'com.android.support:support-annotations:24.0.0',
+    androidPlugin: 'com.android.tools.build:gradle:2.1.3',
+    supportAnnotations: 'com.android.support:support-annotations:24.2.0',
     antlr: "org.antlr:antlr4:$antlrVersion",
     antlrRuntime: "org.antlr:antlr4-runtime:$antlrVersion",
-    javaPoet: 'com.squareup:javapoet:1.5.1',
+    javaPoet: 'com.squareup:javapoet:1.7.0',
```

Comment:
I mean it's fine as long as we don't use any new APIs. Or we could just hold off releasing the next version until AS 2.2 is stable and we say it's required. In the latter case, we can nuke the local jar we use and switch back to a normal dependency.